jdaviz/configs/default/plugins/export/export.py

```diff
@@ -207,6 +220,16 @@ def _set_subset_not_supported_msg(self, msg=None):
         else:  # no subset selected (can be '' instead of None if previous selection made)
             self.subset_invalid_msg = ''
 
+    def _set_dataset_not_supported_msg(self, msg=None):
+        if self.dataset.selected_obj is not None:
+            data_unit = self.dataset.selected_obj.unit
+            if data_unit == u.Unit('DN/s'):

```

Comment:
is there something about this specific data unit that makes it unable to save? just curious what the back story on this is since ive never run into it

Reply:
ahhh i didn't know this! do we want to check if other units produce this warning? is it any sort of combination of units that does this?

Reply:
Not entirely certain as I have only tested against the existing notebooks. Do you have any suggestions on which units I should look out for?

Reply:
This is probably too specific to catch all the edge cases (and we have a lot that snuck into MAST over the years). The cause is that somewhere in the chain, something is doing `this_unit.to_string(format="fits")`. It works in generic format, but not as part of the FITS standard. Do you still have the full traceback? Where exactly is this happening?

Reply:
Maybe we can just try/except the string formatting in advance then?

Reply:
What exactly is the use case that triggers this? "DN" (per second or no) usually is for uncalibrated data. If I understand the context more, then I can hopefully give better options.

Reply:
Depending on where the offending code is, it needs to be more permissive, maybe like this:

```python
>>> u.Unit("DN/s", format="fits", parse_strict="silent").to_string()
'DN/s'
```

Or we can code in exception rule like you have, though one rule per every possible string out there can come back to bite us.

Reply:
We decided to leave this check out and let the snackbar catch this case.  @haticekaratay will create a follow-up ticket to track down and fix the cause.  Thanks all!